### ISSUE TITLE
MariaDB 10.4 users

### ISSUE THREAD

**Issue by lllopo** [phpmyadmin/phpmyadmin#15542]
### Describe the bug

phpMyAdmin is not compatible with MariaDB 10.4 changes to users management ([https://jira.mariadb.org/browse/MDEV-17658](https://jira.mariadb.org/browse/MDEV-17658))
mysql.users table is retired in favor of mysql.global_priv

### To Reproduce
Try to edit user or privileges with MariaDB 10.4

### Server configuration

- Operating system: Windows10
- Web server:Apache 2.4
- Database version: MariaDB 10.4.8
- PHP version: 7.3
- phpMyAdmin version: 4.9.1

**Comment by williamdes** [comment]
@lllopo from my investigation MariaDB provides a view that you can query.

I added and edited an user without any issues

Can you please describe the issue ?

**Comment by Tithugues** [comment]
I do not understand either. It seems that pma doesn't directy rely on `users` table.

**Comment by lllopo** [comment]
It was a test setup that i destroyed since it didnt worked, but will try to set it again and provide more details tomorrow or the day after ... as soon as i have the time for it.

**Comment by lllopo** [comment]
Ok. I re-created the setup and greatly narrowed it down. Not sure if it is a bug anymore ... maybe that's how it should work. Anyway, here's the deal :
I've setup a pristine Maria DB 10.4.8 server with default users. At this point I'm able to login properly (which is expected), BUT when I go to the "user accounts" tab I get "You do not have privileges to manipulate with the users !" and that is my problem. 
And here's the weird (at least for me :-)) thing - I have another phpMyAdmin setup (coming with my WAMP server). I accidentally opened it and it was working ok. So I started comparing what is the difference and switching options to find what is wrong. At the end it turned down to the fact, that I have controluser/controlpass set in my config. Once I removed them, I was able to manipulate the users table  without any problems. At this point I also created the "pma_" tables on the new server and just for the sake of it - brought back the controluser/controlpass ... and this time everything was again ok. 
So, in short - having set controluser/controlpass without the "pma_" tables existing results inability to edit server users privileges. I don't know if this is a bug or by design, so depending on that you can rename or close the issue.

**Comment by williamdes** [comment]
@lllopo you just reproduced #14433 that is now closed on next 4.9 version :)

See: https://github.com/phpmyadmin/phpmyadmin/issues/14433#issuecomment-544249375 to patch your local setup if needed before next version

So if it is a duplicate can you close this issue ?

**Comment by lllopo** [comment]
Yup. Looks like it :-))). Closing. Thank you.
